compiler/rustc_error_codes/src/error_codes/E0724.md

```diff
@@ -1,25 +0,0 @@
-`#[ffi_returns_twice]` was used on something other than a foreign function
```

Comment:
Please revert this change. Error code explanations should never be removed, just updated like you can see in https://doc.rust-lang.org/error_codes/E0729.html (CI will not pass until then in any case).